tests/result/gnutella.pcap.out

```diff
@@ -1,15 +1,17 @@
-Guessed flow protos:	597
```

Comment:
Not sure what triggered that diff..

Reply:
Very strange...I will try to take a look during the weekend

Reply:
@utoni : did you trigger the test on dev?
Anyway... I am seeing the same issue with the current dev. No idea how that is possible.... (and no error yesterday)

Reply:
@IvanNardi 
yeah and it fails now

Reply:
definitely strange since this diff is not related to any external library

Reply:
I just re-run the tests on the original #1698 and now they fail (yesterday everything was fine, obviously)

Reply:
Test now failing also on c0732eda45884de91e0c221e9dd23eeec364bf68, i.e. "HEAD parent"

Reply:
It seems somehow related to one of the LRU caches.

Reply:
I tried up to ab3a678ad423fcd431037093220a145925f64148: all latest commits are now failing on dev
Dev also fails on my notebook, so I think we can exclude some updates/changes on Github runners

Reply:
@utoni , found the issue!! Unbelievable.... PR is coming